server.js

```diff
@@ -1,7 +1,25 @@
+require('dotenv').config();
 var express = require('express')
 var app = express()
+const port = 5000;
+const mongoose = require('mongoose');
+
+const uri = 'mongodb+srv://ishita_naraniya:ishhMongodb@cluster1.ponxmip.mongodb.net/?retryWrites=true&w=majority';
```

Comment:
Move the database connection string and credentials to environment variables to improve security and flexibility. This way, you can easily change the database connection details without modifying the code, and you can avoid exposing sensitive information in your codebase. [important]